atom/browser/renderer_host/atom_render_message_filter.cc

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <stdint.h>
+#include <memory>
+#include <string>
+
+#include "atom/browser/net/preconnect_manager_factory.h"
+#include "atom/browser/renderer_host/atom_render_message_filter.h"
+
+#include "base/bind.h"
+#include "base/bind_helpers.h"
+#include "base/logging.h"
+#include "base/stl_util.h"
+#include "base/task/post_task.h"
+#include "chrome/browser/browser_process.h"
+#include "chrome/browser/chrome_notification_types.h"
+#include "chrome/browser/predictors/preconnect_manager.h"
+#include "chrome/browser/profiles/profile.h"
```

Comment:
We shouldn't be including any `Profile` stuff.

Reply:
removed any include of profile.h, replaced with content::BrowserContext